src/libraries/System.Runtime.InteropServices/gen/Common/Resources/xlf/Strings.cs.xlf

```diff
@@ -844,7 +844,7 @@
       </trans-unit>
       <trans-unit id="MarshallerInNativeReturnPositionMustMatchNativeReturnType">
         <source>All marshallers for values that are passed as the unmanaged return value must have the same unmanaged type.</source>
-        <target state="translated">Všechny zařazovací moduly pro hodnoty, které jsou předány jako nespravovaná návratová hodnota, musí mít stejný nespravovaný typ.</target>
+        <target state="new">All marshallers for values that are passed as the unmanaged return value must have the same unmanaged type.</target>
```

Comment:
@carlossanlop seems like we're still getting loc PRs targeting 9.0 and trying to undo localization.